rust/kernel/init/__private.rs

```diff
@@ -0,0 +1,123 @@
+//! Workaround for specialization
+use super::*;
```

Comment:
```suggestion
// SPDX-License-Identifier: GPL-2.0

//! Workaround for specialization.

use super::*;
```

Similarly in `init.rs` and `pin_data.rs`.